solr/core/src/test/org/apache/solr/metrics/DelegateRegistryTimerTest.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr.metrics;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.codahale.metrics.Clock;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DelegateRegistryTimerTest {
+
+  MetricRegistry.MetricSupplier<Timer> timerSupplier =
+      new MetricSuppliers.DefaultTimerSupplier(null);
+
+  @Test
+  public void update() {
+    DelegateRegistryTimer delegateRegistryTimer =
+        new DelegateRegistryTimer(
+            Clock.defaultClock(), timerSupplier.newMetric(), timerSupplier.newMetric());
+    delegateRegistryTimer.update(Duration.ofSeconds(10));
+    assertEquals(1, delegateRegistryTimer.getPrimaryTimer().getCount());
+    assertEquals(
+        10000000000.0, delegateRegistryTimer.getPrimaryTimer().getSnapshot().getMean(), 0.0);
```

Comment:
These are a bit hard to read, can we pull them into some constants?

Reply:
Tried just making them smaller numbers, no reason to be that large, does that help alleviate the issue?